oak-core/src/main/java/org/apache/jackrabbit/oak/security/user/UserImpl.java

```diff
@@ -148,6 +150,16 @@ public void disable(@Nullable String reason) throws RepositoryException {
         }
     }
 
+    private void canDisableUser() throws RepositoryException {
```

Comment:
nit: I am not super-happy with the name of this method, as it makes me think that it returns a ``boolean``. But I don't have a good name for it either.
(Feel free to ignore this, if you disagree with it.)

Reply:
I'm also not happy with it. Initially I call it `validateCanDisableUser` but again I dislike it.